test/src/helpers.cc

```diff
@@ -65,6 +66,30 @@ void check_partitions(
   }
 
   // Non-empty partitions
+  // TBD: The correctness of this routine (seems in doubt).
```

Comment:
Is this something we need to address before merging this PR?

Reply:
The PR itself is not affected, the concern is for future test cases using it to, perhaps, pass, when they should fail?
As far as I currently know, there are no existing test cases it allows to pass that should fail, 
but *if* the routine is currently in error, there is a possibility something is not being caught.

Reply:
ch8931 created for further future evaluation